Tools/autotest/vehicle_test_suite.py

```diff
@@ -4411,6 +4411,17 @@ def TestLogDownloadMAVProxyNetwork(self, upload_logs=False):
 
         self.set_parameter('SIM_SPEEDUP', 1)
 
+        # ensure the latest log file is very small:
+        self.context_push()
+        self.set_parameter('LOG_DISARMED', 1)
+        self.delay_sim_time(15)
```

Comment:
SIM_SPEEDUP=1 after the delay amd after the wait_gps_fix_type

Reply:
Fixed